suite/tests/client-interface/drx-scattergather.templatex

```diff
@@ -34,20 +34,24 @@ Test restoring the mask register upon asynchronous events
 #ifdef X64
 ERROR: expected k0 == 0xffff, but is 0x1
 #endif
+Test updating the mask register upon asynchronous events
+#ifdef X64
```

Comment:
This golden output file seems complex.  I think in the past I already made this suggestion, but I'm repeating it here: in-test checks are nicer in pretty much every way (esp. *much* easier to debug: the failure is pointed at directly, instead of having to figure out which part of the test generates which part of the output).  Some existing tests are a real pain to look at because we used golden output in the past: drcachesim output results tests, TestMemProtChg.  We should try to avoid making the same mistakes on new tests.  Please consider converting this.

Reply:
Ok, I'll convert once the the restore event has been implemented, since I can't convert it into a hard failure right now. SG?

Reply:
SGTM